include/coreir/ir/value.h

```diff
@@ -99,22 +101,6 @@ class Arg : public Value {
   std::string toString() const { return "Arg(" + field + ")";}
 };
 
-//class CPPLambda : public Value {
-//  public :
-//    typedef std::function<Const(Consts,Consts)> LambdaType;
-//    CPPLambda(ValueType* vtype,LambdaType lambda) : Value(VK_CPPLambda), lambda(lambda) {}
-//    static bool classof(const Value* v) {return v->getKind()==VK_CPPLambda;}
-//    static std::shared_ptr<CPPLambda> make(LambdaType lambda) {
-//      return std::make_shared<CPPLambda>(lambda);
-//    } 
-//  private :
-//    LambdaType lambda;
-//  public :
-//    bool operator==(const Value& r) const {
-//      return false;
-//    }
-//};
```

Comment:
Should this be removed? You removed `VK_CPPLambda` from the enum list, but you kept this around. Should either keep both commented out for consistency (so you don't forget it if you're going to add it back in), or just removed both of them.